build_production/docs/collections-pagination/index.html

```diff
@@ -102,19 +88,33 @@ <h2>Pagination</h2>
 </blockquote>
 <p>Once the <code>pagination</code> has been defined in the header, the template will have access to a special <code>$pagination</code> variable, which has several attributes:</p>
 <ul>
-<li><code>$pagination-&gt;items</code> contains an array of collection items for the current page</li>
-<li><code>$pagination-&gt;currentPage</code> contains the page number of the current page</li>
-<li><code>$pagination-&gt;totalPages</code> contains the total number of pages</li>
+<li>
```

Comment:
@damiani do you know why this file is building with paragraphs for Jason but not for whichever of us built this last?

Reply:
Parsing changes in ParsedownExtra, I believe.